src/settings/SettingsModal.tsx

```diff
@@ -83,8 +83,11 @@ export const SettingsModal: FC<Props> = ({
 
   const devices = useMediaDevices();
   useMediaDeviceNames(devices, open);
-
   const [soundVolume, setSoundVolume] = useSetting(soundEffectVolumeSetting);
+  const [soundVolumeRaw, setSoundVolumeRaw] = useState(soundVolume);
+
+  // Debounce saving the sound volume as it triggers certain components to reload.
+  useEffect(() => {});
```

Comment:
I read the comment, then I look at the line below and I don't get how it achieves this? What am I missing?

Reply:
Ah, I started writing this code and then found `onValueCommit` :)